cpptod.dd

```diff
@@ -829,6 +831,106 @@ void test()
 }
 ------
 
+<hr>$(COMMENT  -------------------------------------------- )
+
+$(H3 <a name="functioninputparameters">Function Input Parameters</a>)
+
+$(H4 The C++ Way)
+
+	$(D C++ does not support explicit input parameters.) If you use a
```

Comment:
`$(D ..)` marks code D inline code:

![image](https://user-images.githubusercontent.com/4370550/28540055-3e36c526-70b4-11e7-8844-8d5e5a1838ca.png)

Did you mean to use `$(B ...)`?